pharo-repository/AWS-Lambda/AWSLambdaConfig.class.st

```diff
@@ -0,0 +1,34 @@
+Class {
+	#name : #AWSLambdaConfig,
+	#superclass : #AWSConfig,
+	#classInstVars : [
+		'default'
+	],
+	#category : #'AWS-Lambda'
+}
+
+{ #category : #'instance creation' }
+AWSLambdaConfig class >> default [
+	default ifNil: [ default := self new ].
+	^default
+]
+
+{ #category : #'class initialization' }
+AWSLambdaConfig class >> initialize [
+	default := nil.
+]
+
+{ #category : #defaults }
+AWSLambdaConfig >> defaultServiceName [
+	^ 'lambda'
+]
+
+{ #category : #accessing }
+AWSLambdaConfig >> hostUrl [
```

Comment:
https://github.com/newapplesho/aws-sdk-smalltalk/blob/master/pharo-repository/AWS-Core/AWSConfig.class.st#L87

Reply:
indeed. 
I removed it.